pkg/apis/addtoscheme_mapi_v1beta1.go

```diff
@@ -0,0 +1,32 @@
+package apis
+
+import (
+	"github.com/openshift/machine-api-operator/pkg/apis/machine"
+	mapiv1 "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
```

Comment:
Please use `mapi`

Reply:
There are some v2 betas as well for some kube resource. So, I think v1 alias makes us future proof

Reply:
I know that and if we move to `v2beta` that we don't need to change many lines of code but just the import line if things are backward compatible. Of course if there are breaking changes we will need to change the line of code in question.